packages/auth/src/mfa/assertions/totp.test.ts

```diff
@@ -127,6 +131,7 @@ describe('core/mfa/assertions/totp/TotpMultiFactorGenerator', () => {
 
 describe('core/mfa/totp/assertions/TotpMultiFactorAssertionImpl', () => {
   let auth: TestAuth;
+  let user: UserInternal;
```

Comment:
We should initialize this in line beforeEach() block just like we initialize auth.